cypress/e2e/swap.test.ts

```diff
@@ -330,12 +330,12 @@ describe('Swap', () => {
           cy.get('#swap-currency-output .token-amount-input').should('not.equal', '')
           cy.get('#swap-button').click()
           cy.get('#confirm-swap-or-send').click()
-          cy.get(getTestSelector('dismiss-tx-confirmation')).click()
+          cy.get(getTestSelector('[data-cy="confirmation-close-icon"]')).click()
```

Comment:
don't use `data-cy`, use `getTestSelector` for `data-testid`s. If you're running up against something already using `data-cy`, swap it out for `data-testid`.